services/user.go

```
@@ -183,6 +183,32 @@ func UpdateEmailConfirmed(id int, db *sql.DB) error {
 	return nil
 }
 
+// UpdatePasswordByID updates a user's password given a user's ID
+func UpdatePasswordByID(id int, p string, db *sql.DB) (models.User, error) {
```

This function should just return an error